src/WebJobs.Extensions.DurableTask/Bindings/BindingHelper.cs

```diff
@@ -86,6 +88,12 @@ private class OrchestrationClientInputData
 
             [JsonProperty("managementUrls")]
             public HttpManagementPayload ManagementUrls { get; set; }
+
+            [JsonProperty("baseUrl")]
+            public string BaseUrl { get; set; }
```

Comment:
Decided to start passing the base url and the required query string parameters, instead of the preconstructed webhook urls. This is a bit more fragile if the webhook urls change, but it has the upside of requiring less work on the extension side in order to make use of new APIs on the JS shim side.